### ISSUE TITLE
Question about the joint prediction

### ISSUE THREAD

**Issue by yxxxxx38324** [MCZhi/GameFormer#2]
Thank you for your amazing work about the interactive prediction and planning！

I have a detail I'd like to consult about joint prediction. 
In joint prediction, is the probability of the final output scene-level, i.e. it has a dimension of 1 * num_mode?
Are joint predictions made in the same way as in DIPP?

I have this question because in The gameformer planner paper, you mentioned that "The model also provides probabilities associated with the different modes of each agent’s trajectory, resulting in tensors of shape N ×6. ", which confuses me a bit, seems to mean that each agent has a different probability output instead of dimension of 1×6.

Or could you please provide the code for the core model of Gameformer? My email is xiaoxiao.yang@sjtu.edu.cn. Thanks a lot!

**Comment by MCZhi** [comment]
Hi, @yxxxxx38324, thank you for your interest in our work.

Regarding your question about joint prediction, our model predicts the probabilities of trajectories for each agent individually. However, during training, we determine the joint best-predicted mode and use this mode to backpropagate the loss.

We are currently preparing for the release and plan to make it available after ICCV's Announcement of decisions, which is scheduled for July 13.

**Comment by yxxxxx38324** [comment]
Does this mean that, during inference, the probability dimension of the output is still N*6, and perhaps the best probability for each agent is not necessarily in the same mode? 
If this is the case, the lack of scene-level probabilities does not seem to predict future trajectories in a scene-consistent manner?

**Comment by MCZhi** [comment]
Yes, during inference, the probability dimension of the output is still N*6. But, we observe that mostly the best probability for each agent is in the same mode.
